iOS Client/ArticlesController/MBArticlesViewController.swift

```diff
@@ -35,6 +38,11 @@ class MBArticlesViewController: UIViewController, UITableViewDelegate, UITableVi
         tableView.delegate = self
         tableView.dataSource = self
         
+        tableView.register(UINib(nibName: "ArticleTableViewCell", bundle: nil), forCellReuseIdentifier: "ArticleTableViewCell")
```

Comment:
Put the cell reuse identifier in a let variable in this file to defend against typos